iac-extensions/docker/src/test/resources/checks/ClearTextProtocolDowngradeCheck/wget.dockerfile

```diff
@@ -15,6 +17,20 @@ RUN wget https://might-redirect.example.com/install.sh --max-redirect=0 -q -O -
 
 RUN wget https://might-redirect.example.com/install.sh -q --max-redirect=0 -O - | sh
 
+# Noncompliant@+1
+RUN wget "https://cmake.org/files/v3.7/cmake-3.7.2-SHA-256.txt.asc"
```

Comment:
I'm a little bit confused 😕 . There is a change in logic for `curl`, but tests are changed for `wget`. Is it expected?

Reply:
Ah, the changed files don't paint the best picture.
If you look at  `SENSITIVE_HTTPS_URL_BEGINNING`, it's changed to include the `ignoreQuotes` predicate. This is used further down in the `wget` commands, which remain unchanged because the name stayed the same.
The `curl` change didn't change the logic but combined the Predicate creation of `EQUALS_PROTO_FLAG_OPTION = equalsIgnoreQuotes("=https")` as it was used two times.